app/src/main/kotlin/com/wire/android/ui/common/topappbar/CommonTopAppBarViewModel.kt

```diff
@@ -77,6 +78,16 @@ class CommonTopAppBarViewModel @Inject constructor(
         }
     }
 
+    private fun legalHoldStatusFlow(userId: UserId) = coreLogic.sessionScope(userId) {
+        observeLegalHoldRequest() // TODO combine with legal host status
+            .map { legalHoldRequestResult ->
+                when (legalHoldRequestResult) {
+                    is ObserveLegalHoldRequestUseCaseResult.ObserveLegalHoldRequestAvailable -> LegalHoldStatus.PENDING
+                    else -> LegalHoldStatus.NO_CONSENT
```

Comment:
Should be no_consent same as pending ?  we display the pending banner of legal hold